src/main/kotlin/com/rahul/quora/clone/WebSecurity.kt

```diff
@@ -0,0 +1,27 @@
+package com.rahul.quora.clone
+
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.security.config.annotation.web.builders.HttpSecurity
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity
+import org.springframework.security.web.SecurityFilterChain
+import org.springframework.security.web.access.expression.DefaultWebSecurityExpressionHandler
+
+@Configuration
+@EnableWebSecurity
+class WebSecurity : DefaultWebSecurityExpressionHandler() {
+
+    @Bean
+    open fun securityFilterChain(httpSecurity: HttpSecurity): SecurityFilterChain {
```

Comment:
why this is required?

Reply:
This is not required since I have excluded SpringSecurityAutoConfiguration. I have commented out.